docs-crawler/docs/aggregate_crawler_output.py

```diff
@@ -1,15 +1,28 @@
+# _*_ Coding: utf-8 -*-
+
+# Iterates over the output generated by the Pytorch,
+# TensorFlow and TensorFlow.js crawlers and provided a sorted
+# output of every function.
+
 import json
 import re
 from translations import WORD_TRANSLATIONS, COMMAND_TRANSLATIONS
 
+# Compiler() is the main parent class responsible for
+# all the translation from the respective output files
+# from all the crawlers.
+
 
 class Compiler(object):
-    tf = []
-    tfjs = []
-    torch = []
+    tf = []  # Create empyt array for TensorFlow.
+    tfjs = []  # Create empty array for TensorFlow.js.
+    torch = []  # Create empty array for Pytorch.
+    # Create dictionary for TensorFlow, TensorFlow.js and Pytorch.
     main_map = {'tf': {}, 'tfjs': {}, 'torch': {}}
     base_defs = set()
 
+    # This constructor calls all our data from tesnorFlow, Tensorflow.js
```

Comment:
For function header documentation it would be useful to use the format used by sphinx: https://pythonhosted.org/an_example_pypi_project/sphinx.html#function-definitions